streams/writable-streams/aborting.js

```diff
@@ -13,8 +13,16 @@ const error2 = new Error('error2');
 error2.name = 'error2';
 
 promise_test(t => {
+  let startResolve;
```

Comment:
I'm leaving the main review to @domenic. I'm just adding a couple of drive-by comments to reduce latency before he can get back.

This is a little bit subtle, but the step "Upon fulfillment of startPromise," makes the following steps be executed asynchronously, even if start() itself returned synchronously.

Aside: almost all WritableStream tests were written against the reference implementation, not Chrome's implementation. There's a few that I initially wrote against Chrome's implementation, but I always verified against the reference implementation before upstreaming.